src/seedu/addressbook/data/person/UnitNumber.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+public class UnitNumber {
+
+    public final String value;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.